internal/db/migrations/005_reservations.sql

```diff
@@ -33,3 +33,18 @@ CREATE TABLE aws_reservation_details
   FOREIGN KEY (reservation_id, provider) REFERENCES reservations(id, provider) ON DELETE CASCADE,
   CHECK (provider = provider_type_aws())
 );
+
+CREATE TABLE gcp_reservation_details
+(
+  reservation_id BIGINT NOT NULL,
+  provider INTEGER NOT NULL DEFAULT provider_type_gcp(),
+  -- pubkey_id BIGINT NOT NULL REFERENCES pubkeys(id),
+  pubkey_id TEXT NOT NULL,
```

Comment:
Not a huge fan of this, I would rather see pubkey reference from the day one, but it will do for now.